File: CoreJavaDay2Assignment/Product.java
package com.sonata;

public class Product {
	int proID;
	String proName;
	double proPrice;
	public double tax(double gst)
	{
		return this.proPrice=this.proPrice+gst;
	}
	public static void main(String args[])
	{
		Product p=new Product();//object
		p.proID=34;
		p.proName="varun";
		p.proPrice=34567.99;
		System.out.println(p.tax(456.9));
	}

}

File: Typescript_Assignment/Product.ts
class Product{
    proid:number;
    proname:string;
    proprice:number;

    constructor(i:number,n:string,s:number){
        this.proid=i;
        this.proname=n;
        this.proprice=s;
    }

    display(gst:number){
        console.log("proid:"+this.proid);
        console.log("proname:"+this.proname);
        console.log("proprice:"+this.proprice);
        console.log("toatlprice with gst:"+this.proprice+gst);
        
    }


}
var p=new Product(2,"mobile",67889);
p.display(567);